warehouse/query-core/src/test/java/datawave/query/config/ShardQueryConfigurationTest.java

```diff
@@ -415,12 +417,14 @@ public void testDeepCopyConstructor() {
         Assert.assertEquals(10000, config.getTfAggregationThresholdMs());
 
         // assert seeks
-        Assert.assertEquals(12, other.getFiFieldSeek());
-        Assert.assertEquals(13, other.getFiNextSeek());
-        Assert.assertEquals(14, other.getEventFieldSeek());
-        Assert.assertEquals(15, other.getEventNextSeek());
+        Assert.assertEquals(12, config.getFiFieldSeek());
+        Assert.assertEquals(13, config.getFiNextSeek());
+        Assert.assertEquals(14, config.getEventFieldSeek());
+        Assert.assertEquals(15, config.getEventNextSeek());
         Assert.assertEquals(16, other.getTfFieldSeek());
```

Comment:
should the tffieldseek switch to config as well, or should we keep that the same?

Reply:
yes it should, good catch